praw/__init__.py

```diff
@@ -1173,7 +1173,7 @@ def _url_update(self, url):
                 url = url.replace("user/"+redditor, 'me')
         return url
 
-    @decorators.restrict_access(scope=None, login=True)
+    @decorators.restrict_access(scope='modself', login=True)
```

Comment:
I believe this needs to become

    @decorators.restrict_access(scope='modself', mod=False)

Because having `modself` in the scope tricks PRAW into thinking that you must be a moderator to perform this action, which you do not.